scripts/install.js

```diff
@@ -109,18 +109,68 @@ function checkAndDownloadBinary() {
       return;
     }
 
-    download(sass.getBinaryUrl(), sass.getBinaryPath(), function(err) {
-      if (err) {
-        console.error(err);
-        return;
-      }
+    var tmpPath = getTempPath(sass.getBinaryName());
+    if (!(sass.hasBinary(tmpPath))) { // download and install
+      download(sass.getBinaryUrl(), tmpPath, function(err) {
+        if (err) {
+          console.error(err);
+          return;
+        }
+        console.log('Binary downloaded at', tmpPath);
+        copyBinary(tmpPath, sass.getBinaryPath());
+      });
 
-      console.log('Binary downloaded and installed at', sass.getBinaryPath());
-    });
+    } else { // install only
+      copyBinary(tmpPath, sass.getBinaryPath());
+    }
   });
 }
 
 /**
+ * Find a temp folder for file
+ *
+ * @param {String} binaryName
+ * @returns {string}
+ * @api private
+ */
+
+function getTempPath(binaryName) {
+  var candidateTmpDirs = [
+    process.env.TMPDIR || process.env.TEMP || process.env.npm_config_tmp,
+    '/tmp',
```

Comment:
That's ok, I'd still prefer you use a tested package that abstracts away this concern.